packages/pwa-kit-create-app/assets/templates/@salesforce/retail-react-app/config/default.js.hbs

```diff
@@ -73,6 +73,11 @@ module.exports = {
             // By setting this to true, the Einstein activities generated by the environment will appear
             // in production environment reports
             isProduction: false
+        },
+        // Datacloud api config
+        dataCloudAPI: {
+            appSourceId: '',
+            tenantId: ''
```

Comment:
You might want to surface these values through the generator script like we are doing for einstein [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/294f6ce74fc34e65cedaefa3f0114704be265b29/packages/pwa-kit-create-app/scripts/create-mobify-app.js#L267)